libs/blockscout-service-launcher/src/metrics.rs

```diff
@@ -11,12 +11,13 @@ pub struct Metrics {
 impl Metrics {
     pub fn new(service_name: &str, endpoint: &str) -> Self {
         let registry = prometheus::default_registry();
-        let metrics_middleware =
-            PrometheusMetricsBuilder::new(&(service_name.to_owned() + "_metrics"))
-                .registry(registry.clone())
-                .endpoint(endpoint)
-                .build()
-                .unwrap();
+        let const_labels = HashMap::from([("rust_service".into(), service_name.into())]);
+        let metrics_middleware = PrometheusMetricsBuilder::new("rust_microservices")
```

Comment:
Nit:
```suggestion
        let metrics_middleware = PrometheusMetricsBuilder::new("rust_microservices_metrics")
```
?

Reply:
It's already a metric, don't see a reason why we need to add it
What else could be with this prefix? rust_microservices_???

Reply:
Make sense! Just thought about that from the historic perspective, as we used `metrics` suffix before